api/init/proxy/interceptor.py

```diff
@@ -1,7 +1,9 @@
 import sys
 from graphql.ast import Document
-import proxy.batch as batch  # Called dynamically with getattr pylint: disable=W0611, useless-import-alias
-import proxy.data_source as data_source  # Called dynamically with getattr pylint: disable=W0611, useless-import-alias
+# Called dynamically with getattr pylint: disable=W0611, useless-import-alias
+from proxy import batch
+# Called dynamically with getattr pylint: disable=W0611, useless-import-alias
+import proxy.data_source as data_source
```

Comment:
Any reason why you're doing `from proxy import batch` and `import proxy.data_source as data_source` rather than `from proxy import data_source`?

Reply:
No, I've updated it in ea3bfb001bb88ea6336de9c255ff35ce52ce9759.